Mathlib/Data/ZMod/Basic.lean

```diff
@@ -4,9 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes
 -/
 import Mathlib.Algebra.CharP.Basic
-import Mathlib.RingTheory.Ideal.Operations
-import Mathlib.Data.Fintype.Units
-import Mathlib.Data.Nat.Parity
+import Mathlib.Algebra.Ring.Prod
 import Mathlib.Tactic.FinCases
```

Comment:
I think this can be removed, #12487 confirms this.
```suggestion

Reply:
Presumably because it's imported transitively, because this file does use `fin_cases`